src/CopyToLatest/CopyToLatest.csproj

```diff
@@ -3,12 +3,12 @@
     <TargetFramework>$(CoreSdkTargetFramework)</TargetFramework>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <CopyBuildOutputToPublishDirectory>false</CopyBuildOutputToPublishDirectory>
+    <Channel>$(SourceBranch.Replace('refs/heads/', ''))</Channel>
```

Comment:
@sfoslund ah... double check that works for internal build and MSRC branches too. I forget that

Reply:
The source branch is always prefixed by refs/heads/. And `internal/` branches are excldued from CopyToLatest here:

https://github.com/dotnet/installer/pull/8092/files#diff-89133339ca240a2b8f3c23650daac903R67-R71